site/src/pages/ManagementSettingsPage/CustomRolesPage/CustomRolesPageView.tsx

```diff
@@ -0,0 +1,166 @@
+import type { Interpolation, Theme } from "@emotion/react";
+import AddOutlined from "@mui/icons-material/AddOutlined";
+import KeyboardArrowRight from "@mui/icons-material/KeyboardArrowRight";
+import Button from "@mui/material/Button";
+import Skeleton from "@mui/material/Skeleton";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import type { FC } from "react";
+import { Link as RouterLink, useNavigate } from "react-router-dom";
+import type { Role } from "api/typesGenerated";
+import { ChooseOne, Cond } from "components/Conditionals/ChooseOne";
+import { EmptyState } from "components/EmptyState/EmptyState";
+import { Paywall } from "components/Paywall/Paywall";
+import {
+  TableLoaderSkeleton,
+  TableRowSkeleton,
+} from "components/TableLoader/TableLoader";
+import { useClickableTableRow } from "hooks";
+import { docs } from "utils/docs";
+
+export type CustomRolesPageViewProps = {
+  roles: Role[] | undefined;
+  canAssignOrgRole: boolean;
+  isCustomRolesEnabled: boolean;
+};
+
+export const CustomRolesPageView: FC<CustomRolesPageViewProps> = ({
+  roles,
+  canAssignOrgRole,
+  isCustomRolesEnabled,
+}) => {
+  const isLoading = Boolean(roles === undefined);
```

Comment:
```suggestion
  const isLoading = roles === undefined;
```

this expression is already a boolean

alternatively, you could just do `!roles`